### ISSUE TITLE
clicking on trends doesn't update the range of the corresponding trend

### ISSUE THREAD

**Issue by amilajack** [tehkaiyu/covid-19#13]
When i click on the "week" trend, the chart doesn't update to show results for the last week. Same thing happens for month. Is this the expected behavior?

**Comment by kyh** [comment]
Yeah, this is expected, but we could rerender the chart with just weekly/monthly data
